AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -171,6 +171,9 @@ eatsmarter.de,inside-digital.de,hs.fi,vrt.be,manager-magazin.de,whats-on-netflix
 !
 !
 !
+rp-online.de#%#//scriptlet('trusted-click-element', 'article[data-consent-article] #consentAccept', '', '500')
+! TODO: remove when `trusted-set-cookie` will be available in all AdGuard release verions
+rp-online.de#%#AG_onLoad(() => { var element = document.querySelector('article[data-consent-article] #consentAccept'); if (/\/info\/consent-remake\//.test(location.pathname) && element) { element.click() }})();
```

Comment:
move `TODO` above of the section. Do not devide current group by comments, directives and hints.